src/stdlib_sorting.fypp

```diff
@@ -1,5 +1,10 @@
 #:include "common.fypp"
-#:set IRS_KINDS_TYPES = INT_KINDS_TYPES + REAL_KINDS_TYPES + STRING_KINDS_TYPES
+
```

Comment:
Suggest you add a comment explaining these fypp definitions. Something along the lines of `For better code reuse in fypp, make lists that contain the input types, with each having output types and a separate name prefix for subroutines`.

Reply:
Also another sentence about why this makes sense would be good (noting it greatly reduces the code, yet is not entirely obvious).

Reply:
Done